operations/CSM_product_management/Configure_Non-Compute_Nodes_with_CFS.md

```diff
@@ -182,4 +205,39 @@ Prior to running NCN personalization, gather the following information:
    using the JSON from the step 3.
 
 > **NOTE:** The CSM configuration layer _MUST_ be the first layer in the
-> NCN personalization CFS configuration.
\ No newline at end of file
+> NCN personalization CFS configuration.
+
+<a name="auto_apply_csm_config"></a>
+## Automatically Apply CSM Configuration to NCNs
+
+CSM configuration can automatically be applied to NCNs by running the
+`apply_csm_configuration.sh` script.
+
+```bash
+ncn-m001# /usr/share/doc/csm/scripts/install/configuration/apply_csm_configuration.sh
+```
+
+### Automatic CSM Configuration Steps
+ 
+By default the script will perform the following steps:
+1. Finds the latest installed release version of the CSM product stream.
+1. Finds the latest commit on the release branch of the `csm-config-management` repo.
+1. Creates or updates the `ncn-personalization.json` configuration file.
+1. Finds all nodes in HSM with the `Management` role.
+1. Disables configuration for all NCN nodes.
```

Comment:
Harold dies a little inside every time he sees the phrase "NCN nodes". I suggest changing to just "NCNs".

Reply:
Fixed